docs/index.md

```diff
@@ -33,23 +33,23 @@ Yes, you will. That's exactly what the local survey will allow you to do!
 
 #### Can I have separate city and region censuses for my country?
 
-Yes. For example, for the United States you might want a census for cities and a census for the states, as both cities and states are releasing open data.
+Yes. For example, for the United States you might want a census for cities and a census for the states, as both cities and states are releasing open data. However, you will need a dedicated instance for each, since comparing them in one instance is not supported in the current version of the Survey. 
 
 #### Can I customize the list of datasets that are surveyed?
 
-Yes, you can. The list of datasets is fully customizable. If you want to add more datasets or change the list of datasets for your local census, you can, just [contact us][contact-us].
+Yes, you can. The list of datasets is fully customizable. If you want to add more datasets or change the list of datasets for your local census, you can, just edit the backend spreadsheet. Be aware that the more datasets you add, the more screen space required for them and the more bulky your survey entries might look. Check the [site-admins][site-admins] section for more specific information. 
 
 #### Can I configure basic information (e.g. site name and title)?
 
-Yes, most of the basic census site information is configurable in an easy way.
+Yes, most of the basic census site information is configurable in an easy way. Check the [site-admins](/site-admins) section for more specific information.
 
 #### Can I run a local Census across multiple countries?
 
-Yes, that's possible – though we encourage people to focus on cities or regions within one country.
+Yes, that's possible – though we encourage people to focus on cities or regions within one country. If you want to learn about the state of open data in countries, we recommend you check the [Open Data Index][open-data-index].
```

Comment:
@tlacoyodefrijol - Maybe it would be better to write check the "GLOBAL open data index" to avoid confusion